recipe/meta.yaml

```diff
@@ -21,7 +21,8 @@ build:
 requirements:
   build:
     - python
-    - nodejs
```

Comment:
All that hard work I did to build the recipe from source. 😄 
I'm OK with the change, but just want to give you some rationale why I spent the time doing it in the first place...

Much of my work on conda-forge is about getting recipes which I can make use of for deploying a full stack internally at my organisation. We are mandated to build from source, and in `notebook`'s case, we have also patched it for improved integration within our network. As a result, I end up having to build notebook from source and having this recipe do the same was quite convenient.

That said, there were some complexities anyway (I'm building on an old gcc from RHEL6, so the nodejs build is a mess) so I'm completely 👍  with making use of pypi here.